Dockerfile

```diff
@@ -1,12 +1,18 @@
 FROM alpine:3.10
 
 ARG ARCH=
+ARG DRONE_TAG=
 
 ENV K3S_RELEASE https://github.com/rancher/k3s/releases/download/${DRONE_TAG}/k3s${ARCH}
 ENV K3S_RELEASE_CHECKSUM https://github.com/rancher/k3s/releases/download/${DRONE_TAG}/sha256sum${ARCH}.txt
 
-RUN wget -O /opt/k3s ${K3S_RELEASE}
+RUN apk add -U jq curl
+RUN curl -LO https://storage.googleapis.com/kubernetes-release/release/v1.17.0/bin/linux/amd64/kubectl \
```

Comment:
Should we env var off `v1.17.0`? This may need to change going forward

Reply:
Yup, will fix that